android_app/app/src/main/java/com/health/openscale/core/bluetooth/lib/YunmaiLib.java

```diff
@@ -51,10 +51,6 @@ public float getMuscle(float bodyFat) {
         float muscle;
         muscle = (100.0f - bodyFat) * 0.67f;
 
-        if (sex == 1) {
```

Comment:
IMO, good option is condition change to `if (activityLevel == 5) {`
@oliexdev can you suggest it?

Reply:
I am not sure, it was a long time ago as I reverse engineered it. I don't own anymore this scale, so I can't test it but this error could be related to issue #484 
So does the muscle values are correct if you change it to activityLevel == 5?